datajunction-server/datajunction_server/construction/build_v2.py

```diff
@@ -974,14 +991,19 @@ async def build_ast(  # pylint: disable=too-many-arguments,too-many-locals
         for ref_expr in reference_expressions:
 
             # Try to find a materialized table attached to this node, if one exists.
-            physical_table = cast(
-                Optional[ast.Table],
-                get_table_for_node(
-                    referenced_node,
-                    build_criteria=build_criteria,
-                ),
-            )
+            physical_table = None
+            if use_materialized:
+                print(f"Checking for physical node: {referenced_node.name}")
```

Comment:
Could you use `logger` here instead of having a print statement?

Reply:
oops! good catch, fixed.